package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/run-server.js",
   "scripts": {
     "lint": "eslint .",
-    "migrate:local": "knex migrate:latest --env $SERVICE_ENVIRONMENT --knexfile ./src/dao/knexfile.js",
+    "migrate:local": "sleep 3 && knex migrate:latest --env $SERVICE_ENVIRONMENT --knexfile ./src/dao/knexfile.js",
```

Comment:
@Druotic this is a hack as a temporary fix for the tests timing issue where the database is not completely ready to start accepting connections and we try to run a database migration. The knexfile script has to be synchronous, so we cannot perform any async checks in there. I thought you fixed this a while back, but maybe you didn't.

Reply:
@austinkelleher It was fixed in the windbreaker-deploy repo, but not in the individual repos: https://github.com/windbreaker-io/windbreaker-deploy/pull/5/files